syncmaster/schemas/v1/transfers/file/base.py

```diff
@@ -94,6 +94,9 @@ def validate_file_name_template(cls, value: str) -> str:
         if missing_keys:
             raise ValueError(f"Missing required placeholders: {', '.join(missing_keys)}")
 
+        if "{run_id}" not in value and "{run_created_at}" not in value:
+            raise ValueError("At least one optional placeholder must be present: run_id or run_created_at")
```

Comment:
```suggestion
            raise ValueError("At least one of placeholders must be present: {run_id} or {run_created_at}")
```
If one of them are mandatory, then it's not optional :)